packages/common/statePortal.ts

```diff
@@ -0,0 +1,197 @@
+import { State } from './states'
```

Comment:
This is for voter registration, not VBM signup.  We want to add this information to the popup from the alloy request when a voter isn't registered

Add a link to "You can register online [for Florida here](link)" (replace Florida with the actual state)  and link to the appropriate state page.  When user clicks on link, make it pop up in a new Tab.

Reply:
Thanks, will be working on this now